[AMD] Clean up hipify base module

xw285cornell opened the pull request:
Summary: Clean up the base module namespace and simplify the hipify script.

Test Plan:
CI 
  buck2 build mode/{opt,amd-gpu} //gen_ai/llm_inference/fb/llm:llama_disagg

Reviewed By: aaronenyeshi

Differential Revision: D55815513

jithunnair-amd commented on tools/amd_build/build_amd.py:
```diff
@@ -5,15 +5,20 @@
         os.path.join(
-            __file__, os.path.pardir, os.path.pardir, os.path.pardir, "torch", "utils"
-        )
+            __file__, os.path.pardir, os.path.pardir, os.path.pardir, "torch", "utils")
```
@xw285cornell Shouldn't the closing parenthesis stay in a new line to be consistent with the code/lint style?

xw285cornell committed:
[AMD] Clean up hipify base module (#123633)

Summary:

Clean up the base module namespace and simplify the hipify script.

Test Plan:
CI 
  buck2 build mode/{opt,amd-gpu} //gen_ai/llm_inference/fb/llm:llama_disagg

Reviewed By: aaronenyeshi

Differential Revision: D55815513

xw285cornell commented:
ok I fixed the internal path so I don't need this any more
